float32 plunge_duration
---
bool success